desktop_version/src/ThirdPartyDeps.c

```diff
@@ -0,0 +1,24 @@
+#include <stddef.h>
+
+// Handle third-party dependencies' needs here
+
+void* SDL_malloc(size_t);
```

Comment:
I believe this will have issues due to calling conventions - it probably works coincidentally as a result of it being __cdecl implicitly for C. If we include SDL_stdinc.h it should bring these declarations in without being in too much (still a lot, but it shouldn't add to the build time).

Reply:
Good point - initially I had included the entirety of `SDL.h` here, but moved it to forward declarations to reduce build time. But you do raise a good point. However, from what I understand, isn't it not recommended to include only one SDL sub-header, and you should instead just include all of `SDL.h`?

Reply:
I think for some subheaders they depend on the monolithic SDL.h, but stdinc happens to be a core file that doesn't care about the others, so it should be safe to include directly (FAudio does this, for example).

Reply:
Alright, fair enough. Rebased, fixed, and force-pushed.